spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -57,14 +57,14 @@ describe('document-capture/components/document-capture', () => {
   });
 
   context('mobile', () => {
-    it('starts with introductory step', () => {
+    it('starts with documents step', () => {
       const { getByText } = render(
         <DeviceContext.Provider value={{ isMobile: true }}>
           <DocumentCapture />
         </DeviceContext.Provider>,
       );
 
-      expect(getByText('doc_auth.info.document_capture_intro_acknowledgment')).to.be.ok();
+      expect(getByText('doc_auth.tips.document_capture_header_text')).to.be.ok();
     });
 
```

Comment:
We probably don't need this case at all, since we're not doing anything special for mobile case in the flow order now.